dspace/config/dspace.cfg

```diff
@@ -1455,34 +1455,15 @@ sitemap.engineurls = http://www.google.com/webmasters/sitemaps/ping?sitemap=
 #
 # No known Sitemap 'ping' URL for MSN/Live search
 
-#####  SHERPA/Romeo Integration Settings ####
-# the SHERPA/RoMEO endpoint
-sherpa.romeo.url = http://www.sherpa.ac.uk/romeo/api29.php
-
-# to disable the sherpa/romeo integration
-# uncomment the follow line
-# webui.submission.sherparomeo-policy-enabled = false
-
-# please register for a free api access key to get many benefits
-# http://www.sherpa.ac.uk/news/romeoapikeys.htm
-# sherpa.romeo.apikey = YOUR-API-KEY
-
 #####  Authority Control Settings  #####
 #plugin.named.org.dspace.content.authority.ChoiceAuthority = \
 # org.dspace.content.authority.SampleAuthority = Sample, \
-# org.dspace.content.authority.LCNameAuthority = LCNameAuthority, \
-# org.dspace.content.authority.SHERPARoMEOPublisher = SRPublisher, \
-# org.dspace.content.authority.SHERPARoMEOJournalTitle = SRJournalTitle, \
 #  org.dspace.content.authority.SolrAuthority = SolrAuthorAuthority
 
 #Uncomment to enable ORCID authority control
 #plugin.named.org.dspace.content.authority.ChoiceAuthority = \
 #    org.dspace.content.authority.SolrAuthority = SolrAuthorAuthority
 
-# URL of ORCID API
-# Defaults to using the Public API (pub.orcid.org)
-orcid.api.url = https://pub.orcid.org/v2.1
```

Comment:
Same here. Don't remove ORCID configurations from dspace.cfg please.  They should stay here and be referenced from the Spring Beans that need them.